ui/packages/ace/lib/webpack.config.js

```diff
@@ -0,0 +1,316 @@
+const path = require('path')
```

Comment:
I don't think you need these changes anymore.

Reply:
Done.